src/tactic/ui/panel/base_table_layout_wdg.py

```diff
@@ -1276,17 +1289,12 @@ def get_action_wdg(my):
             outer.add(my.view_save_dialog)
 
         outer.add_style("min-width: 750px")
-        #outer.add_style("width: 300px")
-        #outer.add_style("overflow: hidden")
-        #outer.add_class("spt_resizable")
-
-        #div.add_style("min-width: 800px")
         div.add_style("height: %s" % height)
         div.add_style("margin: 0px -1px 0px -1px")
 
         
         
-
```

Comment:
Sure ... this was included when our icons had color and we heavily used hidden row.  The shelf lit everything up ... with the new glyph icons, I think this isn't necessary anymore.  Made a comment stating this.